swapserverrpc/server.proto

```diff
@@ -218,6 +228,9 @@ message ServerLoopInRequest {
 
 message ServerLoopInResponse {
     bytes receiver_key = 1;
+
+    bytes receiver_internal_key = 9;
```

Comment:
nit: explicitely state if this is pub or privkey

Reply:
done